docs/sqs.md

```diff
@@ -0,0 +1,112 @@
+---
+id: sqs
+title: AWS SQS
+---
+
+## Introduction
+
+_Amazon Simple Queue Service (SQS)_  is a managed message queue service offered by Amazon Web Services (AWS).
+It provides an HTTP API over which applications can submit items into and read items out of a queue.
+The queue itself is fully managed by AWS, which makes SQS an easy solution
+for passing messages between different parts of software systems that run in the cloud.  
+
+## Dependency
+
+Add the following dependency to get started:
+```scala 
+libraryDependencies += "io.monix" %% "monix-sqs" % "0.2.0"
```

Comment:
@Avasil I guess a new connectors deserves minor version, agree?

Reply:
Usually yes, although with `0.x.y` minor is a bit like a major that it can break binary compatibility (see circe) so there is less discipline